### ISSUE TITLE
Issue creating object where property references object with no owner

### ISSUE THREAD

**Issue by fingask** [afcmrp/python-mfiles#3]
Having an issue using **create_object** when creating a new object with properties specified if the object referenced in a property doesn't have an owner set.

This seems to be related to **get_value_id** checking for a matching **owner_id** on the property being added before it will return an id:

```
  correct_owner = item["OwnerID"] in owner_ids
            if same_name and correct_owner:
                return item["ID"]
```
In this instance the property being added references an object that does not have an owner so I am unable to create the new object.

Object referenced:
```
        {
            "Name": "Tullow Ghana Ltd - (TLW)-(C00340)",
            "ID": 27,
            "ParentID": 0,
            "OwnerID": 0,
            "HasParent": false,
            "HasOwner": false,
            "ValueListID": 105,
            "DisplayID": "27",
            "HasIcon": false,
            "HasAutomaticPermission": false,
            "IsDeleted": false,
            "ItemGUID": "{8DF1EC33-16F4-4BD4-956C-8D6FED2B4F53}"
        }
```

I'm new to the m-files web api so my apologies if I am missing something in my reasoning?

**Comment by afcmrp** [comment]
Good catch, I never ran into that situation when testing. I created the branch owner-unset to try to solve it. I have no way of testing it though so feel free to try it out and make modifications as you see fit!

**Comment by afcmrp** [comment]
I think this is solved by #5 and release 0.5.2. Let me know if you have any issues.
